### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by CRYPTXMNESIA** [WilliamVenner/gmpublisher#152] [Help]
How do I open it in Linux? I'm on Ubuntu 20.04

**Comment by kitrodian** [comment]
I'm not even sure how to install it/make it on Linux, there aren't any clear instructions, at least not that I can find. Can we have some help here?

**Comment by tredI9100** [comment]
yeah, how *do* i use this program on Linux?

**Comment by munoida** [comment]
Okay okay, wherever you may have GMPublisher, depending on what desktop you use, you can right click on it, and click on properties. Head to the permissions tab and then click "Allow executing as program." This should let you launch it now.

However if it doesn't launch, I'm not sure how to help you, because I am having the same issue right now.

**Comment by minihoot** [comment]
try wine or playonlinux

**Comment by Username-Cant-Be-Blank** [comment]
seriously can we have some build instructions on the readme at least? it is the bare minimum
